src/styles/card.css

```diff
@@ -0,0 +1,73 @@
+.product-card {
+  font-family: "Roboto", sans-serif;
+  background-color: #fff;
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+  margin: 16px 3px 0;
+}
+
+.product-card__name {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: var(--main-accent);
+  margin-top: 40px;
+}
+
+.product-card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary);
+  margin-top: 4px;
+}
+
+.stars-container {
+  margin-top: 15px;
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 24px;
+}
+
+.product-card__price span:nth-child(1) {
+  font-size: 12px;
+  color: var(--secondary);
+}
+
+.product-card__price span:nth-child(2) {
```

Comment:
nope, don't increase specificity by using tag selectors, better create elements for each of those